tests/mockData.js

```diff
@@ -31,6 +31,24 @@ const testDate = new Date(TEST_SESSION_DATETIME);
 export const mockDate = new Date(testDate.setMilliseconds(testDate.getMilliseconds() + 900));
 
 export const defaultPassword = 'mysecretpassword!123';
+export const defaultCreationDate = '2019-01-01T06:25:00.000Z';
```

Comment:
sorry for being picky, 01-01 is not a good mock date, can we get a day number greater than 12?

Reply:
we could, but I'm quite sure that would cause quite a few more changes for assertions on rendered content etc. ...